server/graphql/v2/object/TransferWise.ts

```diff
@@ -17,6 +18,7 @@ const TransferWiseFieldGroup = new GraphQLObjectType({
     name: { type: GraphQLString },
     type: { type: GraphQLString },
     required: { type: GraphQLBoolean },
+    refreshRequirementsOnChange: { type: GraphQLBoolean },
```

Comment:
I don't see this field being used, is it a leftover?

Reply:
It is being used here:
https://github.com/opencollective/opencollective-frontend/pull/4441/files#diff-5790329980938a42b82c77bdd4bf5ef1R109

Reply:
Got it! I misread the diff and thought it was part of the `TransferWiseRequiredField` inputs